Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -102,7 +102,10 @@ public Startup(IConfiguration configuration, IWebHostEnvironment env)
     // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
     public void ConfigureServices(IServiceCollection services)
     {
-        services.AddApplicationInsightsTelemetry();
+        services.AddApplicationInsightsTelemetry(x =>
```

Comment:
we prob should ensure we are consistent with app insights in the other modules and use the conn string instead of key

Reply:
are you suggesting we add application insights to the serverless operations and resource provisioner?

Reply:
rather thinking that we kill the key from TF and only rely on conn string

Reply:
ah I see, yes the TF needs to be updated to set the connection string to `ApplicationInsights:ConnectionString`